balancer-js/src/modules/simulation/simulation.module.ts

```diff
@@ -0,0 +1,124 @@
+import { PoolDataService } from '@balancer-labs/sor';
+import { defaultAbiCoder } from '@ethersproject/abi';
+import TenderlyHelper from '@/lib/utils/tenderlyHelper';
+import { BalancerNetworkConfig } from '@/types';
+import {
+  VaultModel,
+  Requests,
+  ActionType,
+} from '../vaultModel/vaultModel.module';
+import { getPoolAddress } from '@/pool-utils';
+import { Zero } from '@ethersproject/constants';
+import { JsonRpcSigner } from '@ethersproject/providers';
+
+export enum SimulationType {
+  Tenderly,
+  VaultModel,
+  Static,
+}
+
+export class Simulation {
```

Comment:
Could probably benefit from some docs/comments to explain what this is for.

Reply:
I added a description to the Simulation module on the generalisedExit PR and only realized after pushing the changes. Please let me know what you think while reviewing it 🙏